extensions/sql/transfer-process-store-sql/src/main/java/org/eclipse/dataspaceconnector/sql/transferprocess/store/PostgresStatements.java

```diff
@@ -44,15 +44,15 @@ public String getFindLeaseByEntityTemplate() {
 
     @Override
     public String getInsertStatement() {
-        return format("INSERT INTO %s (%s, %s, %s, %s, %s, %s, %s,%s, %s) VALUES (?, ?, ?,?, ?, ?, ?, ?, ?);",
+        return format("INSERT INTO %s (%s, %s, %s, %s, %s, %s, %s, %s, %s, %s) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?);",
                 getTableName(), getIdColumn(), getStateColumn(), getStateCountColumn(), getStateTimestampColumn(),
                 getTraceContextColumn(), getErrorDetailColumn(), getResourceManifestColumn(),
-                getProvisionedResourcesetColumn(), getTypeColumn());
+                getProvisionedResourcesetColumn(), getContentDataAddressColumn(), getTypeColumn());
     }
 
     @Override
     public String getFindByIdStatement() {
-        return "SELECT * FROM edc_transfer_process LEFT OUTER JOIN edc_data_request dr ON edc_transfer_process.id = dr.transfer_process_id WHERE edc_transfer_process.id=?";
+        return "SELECT *, dr.id as edc_data_request_id FROM edc_transfer_process LEFT OUTER JOIN edc_data_request dr ON edc_transfer_process.id = dr.transfer_process_id WHERE edc_transfer_process.id=?";
```

Comment:
is there a type in the query?

Reply:
@DominikPinsel no, the query is correct, was done because the postgres jdbc can't handle fields with the same name in the same query, so with only an `*` it will return a result set with two `id` columns, without the alias, this is just a (working :) ) workaround

Reply:
ah sorry :) I didn't read the query correct